evm/src/executor/inspector/cheatcodes/env.rs

```diff
@@ -204,10 +204,18 @@ pub fn apply<DB: DatabaseExt>(
             Bytes::new()
         }
         HEVMCalls::Difficulty(inner) => {
+            ensure!(
+                data.env.cfg.spec_id < SpecId::MERGE,
+                "Difficulty is not supported after the merge. Please use vm.prevrandao instead."
```

Comment:
We may want to reference the hard fork name (Paris) and EIP (https://eips.ethereum.org/EIPS/eip-4399) here

Reply:
Sweet, added the name and a link to it. Did not add a newline as this error will appear just like if it was a revert:

![Screen Shot 2023-05-09 at 13 26 46](https://github.com/foundry-rs/foundry/assets/26014927/c656abbc-eb68-4e3f-8ec6-c06274e817a0)